morpheus/service/vdb/milvus_vector_db_service.py

```diff
@@ -1064,6 +1064,25 @@ def describe(self, name: str, **kwargs: dict[str, typing.Any]) -> dict:
 
         return resource.describe(**kwargs)
 
+    def release_resource(self, name: str) -> bool:
+        """
+        Release a loaded collection from memory.
+
+        Parameters
+        ----------
+        name : str
+            Name of the collection to release.
+
+        Returns
+        -------
+        bool
+            Returns True if release process is successful, otherwise raises MilvusException.
+        """
+
+        self._client.release_collection(collection_name=name)
+
+        return True
```

Comment:
Question: Why does this return a `bool` if it will always be `True`?

Reply:
The `release_collection` function from Milvus is not returning any response. Only it throws `MilvusException` if there is any error while releasing the collection from the memory. And also there isn't an API to identify if a collection is loaded/released from the memory. For testing purpose I have added a return statement. 

I have also tried to capture `MilvusException` from the `release_collection` function by passing a collection that does not exists at all, but it is not raising any error.

Reply:
I think it's more appropriate if this function returns nothing. I also think it's fine if we skip the test that tries to reproduce an error case, since we can't reproduce it.

Reply:
Removed return statement.